extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -90,28 +93,72 @@ export class AmpAdExit extends AMP.BaseElement {
    * @return {function(string): string}
    */
   getUrlVariableRewriter_(args, event, target) {
-    const vars = {
+    const substitutionFunctions = {
       'CLICK_X': () => event.clientX,
       'CLICK_Y': () => event.clientY,
     };
+    const replacements = Services.urlReplacementsForDoc(this.getAmpDoc());
     const whitelist = {
       'RANDOM': true,
       'CLICK_X': true,
       'CLICK_Y': true,
     };
     if (target.vars) {
-      for (const customVar in target.vars) {
-        if (customVar[0] == '_') {
-          vars[customVar] = () =>
-              args[customVar] == undefined ?
-                target.vars[customVar].defaultValue : args[customVar];
-          whitelist[customVar] = true;
+      for (const customVarName in target.vars) {
+        if (customVarName[0] == '_') {
```

Comment:
Consider inverting with continues to save indentation (makes code more readable IMO):
let customVar
if (customVarName[0] != '_' || !(customVar =  /** @type {!./config.Variable} */ (target.vars[customVarName]))) {
   continue;
}
substitutionFunctions[customVarName] = () => { ..

Reply:
Done